plugins/connection/httpapi.py

```diff
@@ -169,37 +179,41 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         )
 
         self._auth = None
-
         if self._network_os:
+          self.load_platfrom_plugins(self._network_os)
 
-            self.httpapi = httpapi_loader.get(self._network_os, self)
-            if self.httpapi:
-                self._sub_plugin = {
-                    "type": "httpapi",
-                    "name": self.httpapi._load_name,
-                    "obj": self.httpapi,
-                }
-                self.queue_message(
-                    "vvvv",
-                    "loaded API plugin %s from path %s for network_os %s"
-                    % (
-                        self.httpapi._load_name,
-                        self.httpapi._original_path,
-                        self._network_os,
-                    ),
-                )
-            else:
-                raise AnsibleConnectionFailure(
-                    "unable to load API plugin for network_os %s"
-                    % self._network_os
-                )
+    def load_platfrom_plugins(self, platform_type=None):
```

Comment:
```suggestion
    def load_platform_plugins(self, platform_type=None):
```
And of course all the things that use the misspelled name

Reply:
Fixed